models/meshmodel/core/policies/rego_policy_relationship.go

```diff
@@ -26,7 +28,10 @@ func NewRegoInstance(policyDir string, regManager *registry.RegistryManager) (*R
 	var store storage.Store
 
 	ctx := context.Background()
-	registeredRelationships, _, _ := regManager.GetEntities(&v1alpha1.RelationshipFilter{})
+	registeredRelationships, _, _, err := regManager.GetEntities(&v1alpha2.RelationshipFilter{})
```

Comment:
This is a concern separate from v1beta1 for Components. 

Relationships v1alpha2 was to have been completely previously, right? Should this go forward independently?

Reply:
Yes, v1alpha2 was completed, but the update was done inplace, i.e. v1alpha1 struct were simply updated in place.

Reply:
If that work was done well and was completed, would we expect that the only change necessary here be that of a swap from “1 “to “2”?